ldn-plugin.js

```diff
@@ -204,34 +207,62 @@ var Ldn = {
      //var windowObject = myMiradorInstance.saveController.getWindowObjectById(_this.data.id)
    //this is an odd way to get the window object, but the above doesn't seem to work
    slotAddress = _this.data.slotAddress;
-   var windowObject = {}
+   var windowObject = {};
    for (var i = 0; i < _this.data.state.slots.length; i++){
      if (_this.data.state.slots[i].layoutAddress === slotAddress){
-       windowObject = _this.data.state.slots[i].window
+       windowObject = _this.data.state.slots[i].window;
      }
    }
    if (windowObject.sidePanelVisible === false){
       windowObject.sidePanelVisibility(true, '0.4s');
    }
   },
+
   parseLayers: function(data){
     var _this = this;
-    var canvasListObject = {}
+    var canvasListObject = {};
     data.otherContent.forEach(function(entry){
-      var canvasId = entry["sc:forCanvas"]
-      var listId = entry["@id"]
+      var canvasId = entry["sc:forCanvas"];
+      var listId = entry["@id"];
       canvasListObject[canvasId] = listId;
     });
 
-    var canvases = _this.data.manifest.jsonLd.sequences[0].canvases
+    var canvases = _this.data.manifest.jsonLd.sequences[0].canvases;
     for (i = 0; i < canvases.length; i++){
-      var canvasId = canvases[i]["@id"]
-      var listId = canvasListObject[canvasId]
+      var canvasId = canvases[i]["@id"];
+      var listId = canvasListObject[canvasId];
       var otherContent = {"@id": listId, "@type": "sc:AnnotationList" };
-      //clear array, since Mirador only supports what attached list
-      canvases[i].otherContent = []
+      if(!canvases[i].otherContent) canvases[i].otherContent = [];
       canvases[i].otherContent.push(otherContent);
     }
+
+    // ^cubap - Is this how you are carrying the myMiradorInstance?
```

Comment:
I don't think this working, it might have to be something more like:  (See the pattern in parseRanges)

new Mirador.AnnotationsTab({
      manifest: _this.data.manifest,
      windowId: this.data.windowId,
      appendTo: _this.data.appendTo.find('.tabContentArea'),
      state: _this.data.state,
      eventEmitter: _this.data.eventEmitter
    });
  },

But i still can get this to work in the same way that it works in parse ranges.